Resources/config/listener.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    cachetool_cache_clearer:
+        class: CacheTool\Bundle\Listener\CacheClearerListener
+        arguments:
+            - @cachetool.cache_clearer.collection
```

Comment:
In this case it violates SRP
and make it harder for testing
Listener shouldn't not about the cache implementations it is triggering